core/threaded/reactor_threaded.c

```diff
@@ -1239,6 +1233,10 @@ int lf_reactor_c_main(int argc, const char* argv[]) {
             LF_PRINT_LOG("---- All worker threads exited successfully.");
         }
     }
+    _lf_normal_termination = true;
+    // Invoke termination function here before freeing the local RTI.
+    termination();
```

Comment:
Yes, I think this is harmless, but it would be cleaner to move the freeing of the local RTI to the termination function. I've done that in commit #73ea7d3524789ca79f3dc143a8d1b8b6760f6e33.